chain-impl-mockchain/src/testing/e2e/fees/mod.rs

```diff
@@ -182,3 +182,91 @@ pub fn owner_delegates_fee() {
 
     ledger_verifier.total_value_is(&Value(expected_total_funds_after));
 }
+
+#[test]
+/// Verifies that after a transaction in a ledger without fees, the total funds do not change and
+/// the fee pots remain empty.
+fn transaction_without_fees() {
+    const ALICE: &str = "Alice";
```

Comment:
This is my mistake, done in some earlier PR, but you can fix it in this one. Please consider moving ALICE and BOB consts to the module level and use it in all tests in current module